README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
Hm, that does not sound right... If you're using the `package.json` descriptor, then I'd use suggest using the npm package instead (https://www.npmjs.com/package/scriptjs). To register in Bower it should've had a `bower.json` at the root as mentioned in their [docs](https://bower.io/docs/creating-packages/#register). Somehow they made it, but it does not sound it's correctly configured, as a Bower component must always have the `bower.json` manifest. In fact you don't need package.json for a bower-only component. `.bower.json` extends the manifest, the same way npm extends the package.json with installation information. The fact that it only exists for installed packages is the reason it's preferred. Do you have more dependencies like that?